From 28290d79f09e96e6c66cef5fd08af29067de62ef Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Thu, 29 Apr 2021 12:06:34 -0700
Subject: [PATCH] Always bundle icons on RHEL 8

The tango-icon-theme package is not currently available for RHEL 8.

Signed-off-by: Scott K Logan <logans@cottsay.net>
---
 package.xml                              | 2 --
 rpm/ros-galactic-tango-icons-vendor.spec | 1 +
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/package.xml b/package.xml
index 49bcb74..2c0fb57 100644
--- a/package.xml
+++ b/package.xml
@@ -14,8 +14,6 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <exec_depend>tango-icon-theme</exec_depend>
-
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
 
diff --git a/rpm/ros-galactic-tango-icons-vendor.spec b/rpm/ros-galactic-tango-icons-vendor.spec
index 91d5222..45a7c10 100644
--- a/rpm/ros-galactic-tango-icons-vendor.spec
+++ b/rpm/ros-galactic-tango-icons-vendor.spec
@@ -46,6 +46,7 @@ mkdir -p obj-%{_target_platform} && cd obj-%{_target_platform}
     -DAMENT_PREFIX_PATH="/opt/ros/galactic" \
     -DCMAKE_PREFIX_PATH="/opt/ros/galactic" \
     -DSETUPTOOLS_DEB_LAYOUT=OFF \
+    -DINSTALL_TANGO_ICONS=ON \
     ..
 
 %make_build
-- 
2.33.1

